app/static/posts.js

```diff
@@ -13,15 +13,40 @@ async function deletePost(e) {
     const gridItem = await document.getElementById(`post-grid-item-${postId}`);
     gridItem.parentNode.removeChild(gridItem);
 
+
     var modalItem = document.getElementById(`post-modal-${postId}`);
     modalItem.parentNode.removeChild(modalItem)
 
     let postCount = parseInt(document.getElementById("profile-post-count").innerText);
-    let updatedPostCount = postCount-1;
+    let updatedPostCount = postCount - 1;
 
     document.getElementById('profile-post-count').innerText = updatedPostCount;
 
     if (updatedPostCount == 0) {
         document.getElementById("upload-first").classList.remove("invisible")
     }
-}
\ No newline at end of file
+}
+
+async function follow(e) {
```

Comment:
This method has no relation to posts. Since the follow button is a component, this method would be better served implemented within the follow button macro.

Reply:
- moved the follow method to the follow button macro